packages/app/package.json

```diff
@@ -87,6 +87,7 @@
     "cypress": "^10.0.3",
     "dotenv": "^16.0.1",
     "eslint": "^8.17.0",
+    "http-server": "^14.1.1",
```

Comment:
I didn't install it here, to avoid dependencies that are not used often on the app